Add Support For Local Unencrypted TAPI (#1244)
Adyen/adyen-node-api-library: README.md

// Step 6: Make the request
const terminalApiResponse: terminal.TerminalApiResponse = await terminalLocalAPI.request(paymentRequest, securityKey);
```
## Using the Local Terminal API Integration without Encryption (Only on TEST)

> In case you would like to develop the Local Terminal API integration parallel to you encryption implementation, you can opt for the unencrypted version. Be sure to remove any encryption details from the CA terminal config page. 

Reviewer: ```suggestion
If you wish to develop the Local Terminal API integration parallel to your encryption implementation, you can opt for the unencrypted version. Be sure to remove any encryption details from the CA terminal config page. 
```
